site/js/components/ee.js

```diff
@@ -0,0 +1,250 @@
+   * Get started
+   * @return {void}
+   */
+  start() {
+    if (this._started.toString() !== 'true') {
+      this._setStarted(true);
+      console.info('%c%s', 'font-weight: bold; background: #174ea6; color: white; padding: .25em;', this._intro.headline);
+      for (const iterator of this._intro.body) {
+        console.info(iterator);
+      }
+    }
+    console.warn('Hint: for help try: %c%s', 'font-family: monospace; font-weight: bold;', 'm100.help();');
+    this._promptQuestion(this._current);
+  }
+
+  /**
+   * Answer
+   * @param {string|number} value - Submit an answer to a prompted question
+   * @return {void}
+   */
+  answer(value) {
+    if (this._started.toString() !== 'true') {
+      console.error('Hmmm... what are you trying to answer?');
+    } else {
+      this._checkAnswer(this._current, value);
+    }
+  }
+
+  /**
+   * Start over
+   * @return {void}
+   */
+  reset() {
+    this._setStarted(false);
+    this._setCurrent(0);
+    this._setTheme(false);
+  }
+
+  /**
+   * Help me...
+   * @return {void}
+   */
+  help() {
+    console.warn("I'm here to help:");
+    console.info('To get started: %c%s', 'font-weight: bold;', 'm100.start();');
+    console.info('To answer a question: %c%s', 'font-weight: bold;', "m100.answer('value');");
+    console.info('To start over: %c%s', 'font-weight: bold;', 'm100.reset();');
+  }
+
+  /**
+   * Gets the current prompt value from local storage.
+   * @return {number} - current prompt value
+   */
+  _getCurrent() {
+    return parseInt(localStorage.getItem('chromeos-m100-current-prompt'));
+  }
+
+  /**
+   * Sets the current prompt value.
+   * @param {number} value - the current prompt integer value
+   * @return {void}
+   */
+  _setCurrent(value) {
+    localStorage.setItem('chromeos-m100-current-prompt', value);
+    this._current = value;
+  }
+
+  /**
+   * Gets the current quiz start value from local storage.
+   * @return {number} - current quiz start value
+   */
+  _isStarted() {
+    return localStorage.getItem('chromeos-m100-quiz-start');
+  }
+
+  /**
+   * Sets the current quiz start value.
+   * @param {number} value - the current quiz start integer value
+   * @return {void}
+   */
+  _setStarted(value) {
+    localStorage.setItem('chromeos-m100-quiz-start', value);
+    this._started = value;
+  }
+
+  /**
+   * Prompt current question
+   * @param {number} current
+   * @return {void}
+   */
+  _promptQuestion(current = 0) {
+    if (typeof this._prompts[current] !== 'undefined') {
+      const prompts = this._prompts[current];
+      console.info('Question #%s: %c%s', current + 1, 'font-weight: bold;', prompts.question);
+    }
+  }
+
+  /**
+   * Prompt a hint
+   * @param {string} hint
+   * @return {void}
+   */
+  _promptHint(hint) {
+    if (hint) {
+      console.info('Hint: %c%s', 'font-style: italic;', hint);
+    }
+  }
+
+  /**
+   * Prompt reward
+   * @param {boolean} giveYouUp
+   * @return {void}
+   */
+  _promptReward(giveYouUp = false) {
```

Comment:
I'm not seeing an instance where this is called without it being `true`. Do we need to have the param (and follow-up if statement) here?

Reply:
lol. legacy testing. I kept rick-rolling myself...